### PR TITLE
Add pronouns for Directors and NEMs

### PR DESCRIPTION
**Pull request by oibind**
Adding pronouns for directors and NEMs profiles.

I'm terrible at CSS. It's honestly a black art to me.

I played around for over an hour trying to style this better but failed miserably. I am only uploading this as a placeholder and show of intention. 

image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -41,6 +42,8 @@ officers:
     discourse: "bolster"
 
   - name: Dylan Wylie
+    pronouns: "he/him"
+    username: bolster
```
Comment (awflwafl) [inline _config.yml:46]: Oops, you're turning us all into @andrewbolster!
```suggestion
```

File: _config.yml [hunk 2]
```diff
@@ -59,6 +62,8 @@ officers:
     linkedin: "colinmitch"
 
   - name: Artemiy Kondratiev
+    pronouns: "he/him"
+    username: bolster
```
Comment (awflwafl) [inline _config.yml:66]:
```suggestion
```

File: _config.yml [hunk 3]
```diff
@@ -69,6 +74,8 @@ officers:
     github: "awflwafl"
 
   - name: Glenn Davidson
+    pronouns: "he/him"
+    username: bolster
```
Comment (awflwafl) [inline _config.yml:78]:
```suggestion
```

File: _config.yml [hunk 4]
```diff
@@ -81,6 +88,8 @@ officers:
     linkedin: "glennrdavidson"
 
   - name: Phil Weir
+    pronouns: "they/them"
+    username: bolster
```
Comment (awflwafl) [inline _config.yml:92]:
```suggestion
```

**Review by awflwafl — COMMENTED** [review]
This is great, thank you for doing it @oibind! Just need to un-bolster us and it's good to deploy.

**Comment by andrewbolster** [comment]
That's a really elegant solution to this. me gusta! 

Is there anything we need to sign off on until you're ready to promote this out of draft @oibind ?

**Comment by oibind** [comment]
As I said I'm not a fan of the styling but I messed around with the CSS for ages and it just wouldn't comply. 

Aesthetically to me it would look better if the text was smaller.

**Review by awflwafl — APPROVED** [review]

**Comment by awflwafl** [comment]
All good to merge assuming you checked with each person.

**Comment by oibind** [comment]
Yes I asked everyone for their pronouns, expect for @andrewbolster's which I just got from Slack.

**Comment by awflwafl** [comment]
Brilliant!

**Pull request merged**